cql3/statements/select_statement.cc

```diff
@@ -45,6 +45,7 @@
 #include "utils/result.hh"
```

Comment:
Yes, it should be "around tokens in the ring" to be precise, or vnodes.

Reply:
Although, "ring" could be also confusing because what tablets partition is also a ring, we just have many rings now. So perhaps we should stick to vnodes, or "tokens in the global ring"? Other suggestions?

Reply:
I'm for "vnodes" (and "vnode boundaries") vs "tablets" (and "tablet boundaries")